### ISSUE TITLE
V8 cannot be built for iOS nor for Android

### ISSUE THREAD

**Issue by ubamrein** [wasmerio/wasmer#5302]
In the blog post of the [5.0](https://wasmer.io/posts/introducing-wasmer-v5) announcement you say, android and iOS support the v8 backend. Sadly it seems to build neither on Android nor on iOS due to missing source-tar balls of the custom wee-8 images. If it is really a build issue I cannot tell, as we did not further follow down that path but tried getting wasmi/wamr to build for iOS.

Nevertheless, the problematic line can be found in the [build.rs](https://github.com/wasmerio/wasmer/blob/7dee3cb0b896ca488934b51eb2b897c51bb3ce99/lib/api/build.rs#L185) file, where for all android/iOS targets no source can be found.

**Comment by syrusakbary** [comment]
Good point! We didn't enabled it because we haven't tested it, but I believe should be easy to enable v8 in iOS an Android. @xdoardo can probably take a look on this in the next week or so

**Comment by xdoardo** [comment]
I'm already working on custom builds for wee8, so I'll probably come back with an update shortly.

**Comment by nanofuxion** [comment]
@xdoardo are there any updates at this time?

**Comment by syrusakbary** [comment]
Hi, we have improved wee8-custom-builds but Android and iOS are not yet there. It should be easy though... any PRs are welcome!

https://github.com/wasmerio/wee8-custom-builds

**Comment by marxin** [comment]
We're planning to work on that as part of https://github.com/wasmerio/v8-custom-builds/pull/12.
